ntbs-service/DataMigration/ImportValidator.cs

```diff
@@ -177,7 +177,7 @@ public ImportValidator(IImportLogger logger, IReferenceDataRepository referenceD
                 return validationsResults;
             }
 
-            var message = $"Case manager {details.CaseManagerUsername} is not present in NTBS database";
+            var message = $"Case manager assigned to notification is not present in NTBS database";
             validationsResults.Add(new ValidationResult(message, new[] { nameof(details.CaseManagerUsername) }));
```

Comment:
I think we should add a comment above these lines saying that we should have already added the case manager if they don't exist, so this error really shouldn't occur